### PR TITLE
Preparing release v1.47.0

### PR DESCRIPTION
**Pull request by frzifus**
## Which problem is this PR solving?
- Closes https://github.com/jaegertracing/jaeger-operator/issues/2258

## Short description of the changes
-

### CODE REVIEW

File: bundle/manifests/jaeger-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -515,11 +515,11 @@ spec:
   maturity: alpha
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.45.0
+  replaces: jaeger-operator.v1.46.0
```
Comment (frzifus) [inline bundle/manifests/jaeger-operator.clusterserviceversion.yaml:518]: https://github.com/jaegertracing/jaeger-operator/actions/runs/5534600439/jobs/10099768942?pr=2263#step:6:279

The CI expects 1.47.0 for some reason.

Comment (frzifus) [reply bundle/manifests/jaeger-operator.clusterserviceversion.yaml:518]: and
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/jaegertracing_jaeger-operator/2263/pull-ci-jaegertracing-jaeger-operator-main-jaeger-e2e-tests/1679186532214247424#1:build-log.txt%3A22

**Review by rubenvp8510 — APPROVED** [review]
LGTM, we need to investigate why is expecting 1.47.0 on the replacement

**Pull request merged**